webapp/components/form/inputChips.js

```diff
@@ -112,15 +112,11 @@ const InputChips = props => {
 
       {showDropdown && (
         <Dropdown
-          items={dropdownItems}
-          itemsLookupFunction={dropdownItemsLookupFunction}
-          itemKeyProp={itemKeyProp}
-          itemKeyFunction={itemKeyFunction}
-          itemLabelFunction={itemLabelFunction}
-          itemLabelProp={itemLabelProp}
+          items={items.length > 0 ? dropdownItems : dropdownItemsLookupFunction}
```

Comment:
I feel a little bit missing with:

dropdownItemsLookupFunction >

itemsLookupFunction ?  async (value) => rejectSelectedItems(await itemsLookupFunction(value)) : null

itemsLookupFunction -> from props

R.reject((item) => R.includes(item, selection))

Reply:
this must be refactored. this pull request is about Dropdown, not InputChips